packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -114,17 +114,18 @@ export class BabylonViewer extends LitElement {
 
     this[$scene].stopAllAnimations();
 
-    // load hdr directly (the size of cubmap is set to be 256 for all renderers)
+    // load and prefilter HDR texture (the size of cubmap is set to be 256 for
+    // all renderers)
     const environment = new HDRCubeTexture(
-        scenario.lighting, this[$scene], 256, false, false, false);
+        scenario.lighting, this[$scene], 256, false, false, false, true);
     this[$scene].environmentTexture = environment;
-    // rotate both skybox and hdr texture for 180 deg to match other renderers
+    // rotate environment for 90 deg, skybox for 270 deg to match the rotation
+    // in other renderers
     environment.setReflectionTextureMatrix(
-        Matrix.RotationY(Tools.ToRadians(180)));
+        Matrix.RotationY(Tools.ToRadians(90)));
     const skybox =
         this[$scene].createDefaultSkybox(this[$scene].environmentTexture!);
-    skybox!.rotate(Axis.Y, Math.PI, Space.WORLD);
-    skybox!.infiniteDistance = true;
```

Comment:
Since not setting far clip has already solved the skybox disappearing problem, this line is useless.

Reply:
You might want to keep infiniteDistance; ours are infinite and I'm concerned that otherwise the skybox might change based on camera radius (which it shouldn't).